db/dcrsqlite/sync.go

```diff
@@ -405,3 +409,54 @@ func (db *wiredDB) getBlock(ind int64) (*dcrutil.Block, *chainhash.Hash, error)
 
 	return block, blockhash, nil
 }
+
+// ImportSideChains imports all side chains. Similar to pgblockchain.MissingSideChainBlocks
+// plus the rest from main.go
+func (db *wiredDB) ImportSideChains(collector *blockdata.Collector) error {
+	tips, err := rpcutils.SideChains(db.client)
+	if err != nil {
+		return err
+	}
+	hashlist := make([]*chainhash.Hash, 0)
+	for it := range tips {
+		sideHeight := tips[it].Height
+		log.Tracef("Primary DB -> Getting base DB side chain with tip %s at %d.", tips[it].Hash, sideHeight)
+		sideChain, err := rpcutils.SideChainFull(db.client, tips[it].Hash)
+		if err != nil {
+			log.Errorf("Primary DB -> Unable to get side chain blocks for chain tip %s: %v", tips[it].Hash, err)
+			return err
+		}
+		sideHeight -= int64(len(sideChain)) - 1
```

Comment:
The computed value does not appear to be used.

Reply:
Seems like this might be copy pasta, but I want to be sure a detail wasn't left out here.